### PR TITLE
add q1 roadmap items

### PR DESCRIPTION
**Pull request by taylorsperry**
Updating DevX strategy page to include FY23 Q1 roadmap items.

### CODE REVIEW

File: content/strategy-goals/strategy/enablement/dev-experience/index.md [hunk 1]
```diff
@@ -31,35 +31,24 @@ We inherit Sourcegraph's [engineering principles and practices](../../../../depa
...
-### Next (6–12 months)
+#### [Improve engineering onboarding: tooling](https://github.com/sourcegraph/sourcegraph/issues/31005)
+
+#### [Improve support for application developer experience](https://github.com/sourcegraph/sourcegraph/issues/31006)
 
-We are currently drafting opportunities and projects we plan to explore over next year in our [strategy draft document](https://docs.google.com/document/d/1IrIe7NUEr_0RscvWDuvtjAKFRIp4agpc_in1_6T5WBQ/edit#).
+### Next (6–12 months)
 
-### Later
+#### Standardize testing, logging, metrics, and code checks
```
Comment (bobheadxi) [inline content/strategy-goals/strategy/enablement/dev-experience/index.md:52]:
```suggestion
#### [Standardize testing, logging, metrics, and code checks](https://github.com/sourcegraph/sourcegraph/issues/30552)
```

Comment (bobheadxi) [inline content/strategy-goals/strategy/enablement/dev-experience/index.md]:
```suggestion
#### [Standardize logging, metrics, and errors](https://github.com/sourcegraph/sourcegraph/issues/30552)

#### [Standardize code checks and testing](#TODO)
```

File: content/strategy-goals/strategy/enablement/dev-experience/index.md [hunk 2]
```diff
@@ -31,35 +31,24 @@ We inherit Sourcegraph's [engineering principles and practices](../../../../depa
...
-- **DX is observable**
-  - Tactic: Find reliable metrics that we can start monitoring with low prior engineering efforts.
-  - Tactic: Evaluate and decide if we continue with OkayHQ.
-  - OKR: Those metrics are sufficient to corroborate existing DX hypotheses (or invalidate them).
+#### [SOC2 Compliance](https://github.com/sourcegraph/sourcegraph/issues/31008)
 
-### Next (6–12 months)
+#### [Improve engineering onboarding: tooling](https://github.com/sourcegraph/sourcegraph/issues/31005)
+
+#### [Improve support for application developer experience](https://github.com/sourcegraph/sourcegraph/issues/31006)
```
Comment (bobheadxi) [inline content/strategy-goals/strategy/enablement/dev-experience/index.md]: Formatting nit: maybe these could be list items instead?

File: content/strategy-goals/strategy/enablement/dev-experience/index.md (hunk 1 above)
Comment (bobheadxi) [reply content/strategy-goals/strategy/enablement/dev-experience/index.md]: @taylorsperry you were too fast on the other edit 😆 but I think we should split these up into observability standardization (point 1) and dev standardization (point 2 code checks and testing)

Comment (taylorsperry) [reply content/strategy-goals/strategy/enablement/dev-experience/index.md]: Allllllways happy to break up a roadmap item 🥳

File: content/strategy-goals/strategy/enablement/dev-experience/index.md [hunk 3]
```diff
@@ -31,35 +31,26 @@ We inherit Sourcegraph's [engineering principles and practices](../../../../depa
...
+- 🎯 Red builds on the pipeline are caused by legitimate test failures, not the lower level infrastructure they're running on
+- 🎯 Total build time on main is within ~5 minuts of previous deployment model
 
 ## Roadmap
 
-### Now (3 months)
+### FY23 Q1
 
-Our high-level roadmap is blurry; we aim to work on the most impactful problems.
+- #### [Improve CI stability](https://github.com/sourcegraph/sourcegraph/issues/31003)
```
Comment (bobheadxi) [inline content/strategy-goals/strategy/enablement/dev-experience/index.md]:
```suggestion
- [Improve CI stability](https://github.com/sourcegraph/sourcegraph/issues/31003)
```

File: content/strategy-goals/strategy/enablement/dev-experience/index.md [hunk 4]
```diff
@@ -31,35 +31,26 @@ We inherit Sourcegraph's [engineering principles and practices](../../../../depa
...
-  - OKR: Improved NPS, Time To Tenth PR
-- **DX is observable**
-  - Tactic: Find reliable metrics that we can start monitoring with low prior engineering efforts.
-  - Tactic: Evaluate and decide if we continue with OkayHQ.
-  - OKR: Those metrics are sufficient to corroborate existing DX hypotheses (or invalidate them).
+- #### [SOC2 Compliance](https://github.com/sourcegraph/sourcegraph/issues/31008)
+
+- #### [Improve engineering onboarding: tooling](https://github.com/sourcegraph/sourcegraph/issues/31005)
+
+- #### [Improve support for application developer experience](https://github.com/sourcegraph/sourcegraph/issues/31006)
```
Comment (bobheadxi) [inline content/strategy-goals/strategy/enablement/dev-experience/index.md]:
```suggestion
- [SOC2 Compliance](https://github.com/sourcegraph/sourcegraph/issues/31008)

- [Improve engineering onboarding: tooling](https://github.com/sourcegraph/sourcegraph/issues/31005)

- [Improve support for application developer experience](https://github.com/sourcegraph/sourcegraph/issues/31006)
```

File: content/strategy-goals/strategy/enablement/dev-experience/index.md [hunk 5]
```diff
@@ -31,35 +31,26 @@ We inherit Sourcegraph's [engineering principles and practices](../../../../depa
...
 
 ### Next (6–12 months)
 
-We are currently drafting opportunities and projects we plan to explore over next year in our [strategy draft document](https://docs.google.com/document/d/1IrIe7NUEr_0RscvWDuvtjAKFRIp4agpc_in1_6T5WBQ/edit#).
+- #### [Standardize logging, metrics, and errors](https://github.com/sourcegraph/sourcegraph/issues/30552)
```
Comment (bobheadxi) [inline content/strategy-goals/strategy/enablement/dev-experience/index.md]:
```suggestion
- [Standardize logging, metrics, and errors](https://github.com/sourcegraph/sourcegraph/issues/30552)
```

File: content/strategy-goals/strategy/enablement/dev-experience/index.md [hunk 6]
```diff
@@ -31,35 +31,26 @@ We inherit Sourcegraph's [engineering principles and practices](../../../../depa
...
 
-### Later
+- #### Standardize code checks and testing
```
Comment (bobheadxi) [inline content/strategy-goals/strategy/enablement/dev-experience/index.md]:
```suggestion
- Standardize code checks and testing
```

File: content/strategy-goals/strategy/enablement/dev-experience/index.md [hunk 7]
```diff
@@ -31,35 +31,26 @@ We inherit Sourcegraph's [engineering principles and practices](../../../../depa
...
 
-We are currently drafting future bets in our [strategy draft document](https://docs.google.com/document/d/1IrIe7NUEr_0RscvWDuvtjAKFRIp4agpc_in1_6T5WBQ/edit#).
+- #### Improve incident detection on Sourcegraph Cloud
```
Comment (bobheadxi) [inline content/strategy-goals/strategy/enablement/dev-experience/index.md]:
```suggestion
- Improve incident detection on Sourcegraph Cloud
```

**Review by bobheadxi — COMMENTED** [review]
a bunch of list fix recommendations (github wont let me do it all in one go 😠 )

**Pull request merged**